test/spiretest/dir.go

```diff
@@ -0,0 +1,23 @@
+package spiretest
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TempDir creates a temporary directory that is cleaned up when the test
+// finishes.
+// TODO: remove when go1.15 is out, which introduces a new method on
+// *testing.T for this purpose.
```

Comment:
Make sure we open a GitHub issue to track this Todo. Good candidate for a "good first issue" tag too

Reply:
Good call.

Reply:
#1758